### ISSUE TITLE
Avoid refreshing all tabs when they are not open

### ISSUE THREAD

**Issue by jeremistadler** [TablePlus/TablePlus#3456]
Some tables are heavy to just load the COUNT(*) of rows, can the query refresh be deferred to when the user opens the tab?

**Comment by swznd** [comment]
agree, the normal behavior should be like a web browser, just refresh the active tab.

If you still want to refresh all tabs, consider using a different key combination, like ctrl/cmd + shift + r
